core/lib/spree/permission_sets/user_management.rb

```diff
@@ -2,7 +2,16 @@ module Spree
   module PermissionSets
     class UserManagement < PermissionSets::Base
       def activate!
-        can :manage, Spree.user_class
+        can [:admin, :display, :create, :update], Spree.user_class
+
+        # due to how cancancan filters by associations,
+        # we have to define this twice, once for `accessible_by`
+        can :update_email, Spree.user_class, spree_roles: { id: nil }
+        # and once for `can?`
+        can :update_email, Spree.user_class do |user|
+          user.spree_roles.none?
```

Comment:
The only other idea that comes to top of mind for me involves another 'assumption' about usage: namely that we restrict the UserManagement permission to not be able to edit the email of users who have the SuperUser permission, but allow all other editing. This would prevent privilege escalation to SuperUser, but not any other type of privilege escalation.